### PR TITLE
Remove the catalog

### PR DESCRIPTION
**Pull request by houshengbo**
Closes-Bug: #1165

### CODE REVIEW

**Comment by jasonpet** [comment]
You cannot remove util.sh.  This is currently needed by other repositories.

File: catalog/util.sh [hunk 1]
```diff
@@ -1,74 +0,0 @@
-#!/bin/bash
```
Comment (jasonpet) [inline catalog/util.sh]: Removing util.sh will break other repositories during their installs.

**Comment by jasonpet** [comment]
I do not want multiple copies of util.sh in each new repository.  We already ran into an issue where the util.sh from openwhisk repo and the new catalog repo were not in sync.  We should only have one and reference that from the other repositories.

**Comment by jasonpet** [comment]
@houshengbo - we need to do one last check to make sure everything is in sync between the catalog repository and all of these files we are deleting.

**Comment by houshengbo** [comment]
@jasonpet This one is ready as well.

**Comment by jasonpet** [comment]
pg1#419

**Comment by houshengbo** [comment]
@jasonpet The catalog repo has been synchronized with openwhisk catalog, so the catalog under openwhisk is able to remove.

**Pull request merged**